mailpoet/lib/Segments/DynamicSegments/Filters/WooCommerceTotalSpent.php

```diff
@@ -3,52 +3,30 @@
 namespace MailPoet\Segments\DynamicSegments\Filters;
 
 use MailPoet\Entities\DynamicSegmentFilterEntity;
-use MailPoet\Entities\SubscriberEntity;
-use MailPoet\Util\DBCollationChecker;
 use MailPoet\Util\Security;
 use MailPoetVendor\Carbon\Carbon;
 use MailPoetVendor\Doctrine\DBAL\Query\QueryBuilder;
-use MailPoetVendor\Doctrine\ORM\EntityManager;
 
 class WooCommerceTotalSpent implements Filter {
   const ACTION_TOTAL_SPENT = 'totalSpent';
 
-  /** @var EntityManager */
-  private $entityManager;
-
-  /** @var DBCollationChecker */
-  private $collationChecker;
-
   /** @var WooFilterHelper */
   private $wooFilterHelper;
 
   public function __construct(
-    EntityManager $entityManager,
-    DBCollationChecker $collationChecker,
     WooFilterHelper $wooFilterHelper
   ) {
-    $this->entityManager = $entityManager;
-    $this->collationChecker = $collationChecker;
     $this->wooFilterHelper = $wooFilterHelper;
   }
 
   public function apply(QueryBuilder $queryBuilder, DynamicSegmentFilterEntity $filter): QueryBuilder {
-    global $wpdb;
-    $subscribersTable = $this->entityManager->getClassMetadata(SubscriberEntity::class)->getTableName();
     $filterData = $filter->getFilterData();
     $type = $filterData->getParam('total_spent_type');
     $amount = $filterData->getParam('total_spent_amount');
     $days = $filterData->getParam('total_spent_days');
 
     $date = Carbon::now()->subDays($days);
     $parameterSuffix = $filter->getId() ?? Security::generateRandomString();
-    $collation = $this->collationChecker->getCollateIfNeeded(
```

Comment:
Just checking the commit message and the commit changes, it is not clear to me which helper will be used instead and why the removed code is not necessary anymore. Could you please explain?

Reply:
I apologize for the confusing commit! This indeed doesn't make sense. These are changes I should have made in 30c8ce08395d5e51e4e68a656ee02b9460e470cd, when I actually refactored this filter to use the `wooFilterHelper`, at which point `$collation` became an unused variable. 

Would it be best to rewrite the commit message?

Reply:
Thanks for the clarification. I don't have a strong opinion here, but I'm inclined to say it is worth rewriting the commit message to make this more clear.

Reply:
I reworded in ef654550ffc9e0191e3f2ebc6c59d5444aaefa1e